### PR TITLE
Replaced print statements with logging module

### PR DESCRIPTION
**Pull request by PvtHaggard**
Replaced all print statements with the logging module as requested in #10. I also included an example file that demonstrates how to disable the messages.

It is working fine with py 3.5. I am unable to test on py 2.7 right now.

### CODE REVIEW

File: forecastiopy/ForecastIO.py [hunk 1]
```diff
@@ -6,12 +6,16 @@
...
-from __future__ import print_function
 import sys
 import json
 import requests
+import logging
 
 
+log = logging.getLogger(__name__)
+log.setLevel(level=logging.WARNING)
+log.addHandler(logging.NullHandler())
```
Comment (tomschr) [inline forecastiopy/ForecastIO.py]: According to the [Hitchhiker's Guide to Python](http://docs.python-guide.org/en/latest/writing/logging/#logging-in-a-library) I would move the `NullHandler` into the `__init__.py` file:

```python
import logging
logging.getLogger(__name__).addHandler(logging.NullHandler())
```

**Comment by PvtHaggard** [comment]
Right. I've gone with your suggestion and moved the logging initialization to __init__.py.

**Comment by dvdme** [comment]
I'm currently on vacation but I'll merge this next week

**Comment by PvtHaggard** [comment]
I hope you enjoyed your vacation, @dvdme. Are you able to look at my changes and merge them sometime soon, please?

**Comment by PvtHaggard** [comment]
Any update on merging?

**Pull request merged**